pkg/ebpf/manager.go

```diff
@@ -66,23 +66,26 @@ type Modifier interface {
 	fmt.Stringer
 	// BeforeInit is called before the ebpf.Manager.InitWithOptions call
 	// names.ModuleName refers to the name associated with Manager instance.
-	// The io.ReaderAt is a reader for the eBPF bytecode.
-	BeforeInit(*manager.Manager, names.ModuleName, *manager.Options, io.ReaderAt) error
+	BeforeInit(*manager.Manager, names.ModuleName, *manager.Options) error
 
 	// AfterInit is called after the ebpf.Manager.InitWithOptions call
 	AfterInit(*manager.Manager, names.ModuleName, *manager.Options) error
 }
 
 // InitWithOptions is a wrapper around ebpf-manager.Manager.InitWithOptions
 func (m *Manager) InitWithOptions(bytecode io.ReaderAt, opts *manager.Options) error {
+	if err := m.LoadELF(bytecode); err != nil {
```

Comment:
can you add a comment to state why this is necessary **before** calling to all the modifiers` `BeforeInit`?